### DISCUSSION TITLE
Vuetify toasts

### DISCUSSION THREAD

**Discussion by MaximStone** [vuetifyjs/vuetify#14093] [Ideas]
Hello! Thanks for brilliant framework!

I wonder if there a chance to see toasts (notifications) inside Vuetify in nearest future?

I mean something like that https://element.eleme.io/#/en-US/component/notification 

Thanks!

**Comment by pedro-gilmora** [comment]
that's a service you can write by your own. A simple v-for iterating the snackbars and alerts within them inside with different positions, something like 
```
this.$showToast({
  title: string, 
  content: () => VNode[] | string, 
  color: string, actions: ActionOptions[],
  //... more props
})
```
